Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
+xml2rfc = "*"
+id2xml = "*"
+bibtex2rfc = "*"
+
+[requires]
+python_version = "3.8"
```

Comment:
Please do not hard-code the python version

Reply:
@flichtenheld Agreed and fixed

Reply:
> @flichtenheld Agreed and fixed

You didn't update Pipfile.lock, so it will now always be regenerated on pipenv install. Please fix.